packages/uv/package.yaml

```
@@ -0,0 +1,59 @@
+---
+name: uv
+description: |
+  An extremely fast Python package and project manager, written in Rust
+homepage: https://docs.astral.sh/uv/
+licenses:
+  - Apache-2.0
+  - MIT
+languages:
+  - python
```

```suggestion
  - Python
```
That would make the CI/CD happy